tools/debug/README.md

```diff
@@ -0,0 +1,37 @@
+# DebugSymbolViewer
+
+The Debug Symbol viewer will help the developer when a crash occurs.
+It will show some call stack information when the application has faced an abort. To be precise, it will show the symbols corresponding to the stack addresses present in the kernel text address range.
+
+### Prerequisites
+Install python 3.8 & python 2.7, pyserial(at pip)
```

Comment:
Both python 3.8 and 2.7 are required ? 3.8 cannot cover the 2.7?

Reply:
No, Python 3 is *not backward compatible*
This was one of the problems that would arise due to merging of the new tool with ramdumpParser, had already intimated the same.